tackle-test-generator-unit/test/java/org/konveyor/tackle/testgen/TestUtils.java

```diff
@@ -164,8 +164,8 @@ public static class AppUnderTest {
         public String appPath;
         public String appClasspathFilename;
         public String testPlanFilename;
-        
```

Comment:
all empty lines are marked as changed - any idea why?

Reply:
The IDE apparently removed tabs, and in many cases replaces tabs with spaces, on opening the files